GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -0,0 +1,61 @@
+﻿using CommandLine;
+using GVFS.Common;
+using System;
+
+namespace GVFS.CommandLine
+{
+    [Verb(ConfigVerbName, HelpText = "Set and Get GVFS config settings.")]
+    public class ConfigVerb : GVFSVerb
+    {
+        private const string ConfigVerbName = "config";
+        private LocalGVFSConfig localConfig;
+
+        public override string EnlistmentRootPathParameter { get; set; }
+
+        protected override string VerbName
+        {
+            get { return ConfigVerbName; }
+        }
+
+        public override void Execute()
+        {
+            string[] args = Environment.GetCommandLineArgs();
+            if (args.Length < 3 || args.Length > 4)
+            {
+                string usageString = string.Join(
+                    Environment.NewLine,
+                    "Error: wrong number of arguments.",
+                    "Usage: gvfs config <config> <value>");

```

Comment:
This usage message is incorrect, since value is optional and causes a different behavior. Please be more complete here.